prow/plugins/releasenote/releasenote_test.go

```diff
@@ -57,7 +58,7 @@ func TestReleaseNoteComment(t *testing.T) {
 		{
 			name:          "reviewer release-note",
 			action:        "created",
-			isReviewer:    true,
+			isMember:      true,
```

Comment:
As this is checking if the commenter is a member, the name of this test should also be modified